src/stream.rs

```diff
@@ -175,6 +176,32 @@ impl VsockStream {
     }
 }
 
+/// Connection info for a Vsock Stream.
+///
+/// See [`Connected`] for more details.
+///
+#[derive(Debug, Clone)]
```

Comment:
Derive `Eq` and `PartialEq` too if possible, to make it easier for users of the library to write tests involving this type.

Reply:
Agreed, made that change.